### ISSUE TITLE
A11y_Fabric Extensions for VS Code_Fabric_GraphQI ApI _CCA:keyboard focus for GraphQI ApI is having color contrast ratio 1.243:1 which is less than 3:1.

### ISSUE THREAD

**Issue by seungmyang** [microsoft/vscode#205898]
Does this issue occur when all extensions are disabled?: No

- VS Code Version: 1.86.2
- OS Version: Windows 11 Enterprise

**Repro Steps:**
1. Launch Visual Studio Code
2. Install the "Fabric workspaces" and "Fabric App Development" extensions
3. Tab to fabric workspaces icon in left activity bar and press enter
4. Sign in to fabric with appropriate credentials and
5. Tab to command palette and type Fabric create item and
6. Select Graph QI Api preview  and type Graph QI Api
7. Tab to tree view Graph QI Api
8. Tab using keyboard 
9. Turn on the CCA tool.
10. Observe the keyboard focus ratio.

**Actual Result:**
 keyboard focus for Graph QI Api is having less Color contrast ratio.

**Note :**
Similar issues has been observed for Entire screen.

**Expected Result:**
Keyboard focus for Graph QI Api should have more Color contrast ratio.

**User Impact:**
Low Vision users will face difficulties keyboard focus for Graph QI Api is having less Color contrast ratio.

**Comment by meganrogge** [comment]
Is this an issue specific to that extension? If so, pls create an issue for them.

**Comment by seungmyang** [comment]
Hi Megan,

It's related to the global navigation experience in VS Code, not specific to the extension.

@Sunkara Jagan Sai (TATA CONSULTANCY SERVICES ***@***.***> found the bug and requested to open the external bug as it's related to the global navigation of VS Code.

Thanks,
Seung

**Comment by meganrogge** [comment]
Thanks, a screenshot of the particular area would be helpful.

**Comment by meganrogge** [comment]
Also, if the screenshot could show the issue in VS Code without that extension, it would be better.

**Comment by seungmyang** [comment]
@Sunkara Jagan Sai (TATA CONSULTANCY SERVICES ***@***.***>, could you provide a screenshot to show the issue in VS Code without the extension, please?

Thanks,
Seung

**Comment by v-jagansai** [comment]
@seungmyang Issue in VS Code without that extension screen shot
![image](https://github.com/microsoft/vscode/assets/109065315/0d4910e3-0fce-4d92-8593-2196fff4422e)

**Comment by seungmyang** [comment]
@Shahrukh ***@***.***>, where do you see the white text or keyboard focus? Could you add screenshot to the issue #205898 with the Issue in VS Code without that extension, please?

BTW, the global navigation item passed the contrast requirement.
[A screenshot of a computer  Description automatically generated]

**Comment by Samhitha66** [comment]
#A11yMAS; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2023;

**Comment by seungmyang** [comment]
We need updated screenshot of the issue. I don't see the white text with contrast issue from the screenshot Samhitha highlighted.
![image](https://github.com/microsoft/vscode/assets/58715187/a69c8652-5fd8-4b10-96e9-662497b26266)

**Comment by v-jagansai** [comment]
![Screenshot_16](https://github.com/microsoft/vscode/assets/109065315/c036852e-3765-4673-92e0-322b6ce997ee)
Verified the issue , Issue not repro , PFA 
Version: 1.87.2 (user setup)
Commit: 863d2581ecda6849923a2118d93a088b0745d9d6
Date: 2024-03-08T15:20:17.278Z
Electron: 27.3.2
ElectronBuildId: 26836302
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Windows_NT x64 10.0.26085
Feel free to close this bug

**Comment by meganrogge** [comment]
@v-jagansai can this be closed?

**Comment by v-jagansai** [comment]
@meganrogge Could you please expand that and check contrast inside

**Comment by Samhitha66** [comment]
@meganrogge Primary bugs are fixed and closed. Feel free to close this bug aswell
